manifests/prerequisites.pp

```diff
@@ -13,6 +13,7 @@
         # Monit is not available in the standard repositories
         include ::epel
     } elsif $::lsbdistcodename == 'buster' and $manage_backports {
+        include ::apt
```

Comment:
While this could potentially cause issues if "apt" is included elsewhere in the catalog with parameters, this segment can be avoided with $manage_backports = false. So this is good enough. Better than having yet another option that determines if the "apt" class should be included.